packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -40,7 +43,7 @@ const Conversation = WebexPlugin.extend({
    */
   getUrlFromClusterId({cluster, id}) {
     const clusterId =
-      cluster === 'us' ? 'urn:TEAM:us-east-1_int13:identityLookup' : cluster;
+      cluster === 'us' ? DEFAULT_CLUSTER : cluster;
```

Comment:
As an alternative to this, could we, in the case that cluster is undefined in the DTO, set the value to the default cluster as well?

```js
const clusterId = (!cluster || cluster === 'us') ? DEFAULT_CLUSTER : cluster;
```

this could also be done with DTO defaults in the method param's definition.

If this does not work in testing or fails to work in specific cases, this comment can be ignored.